tests/providers/google/cloud/hooks/test_compute.py

```diff
@@ -37,6 +37,420 @@
 GCE_REQUEST_ID = 'request_id'
 GCE_INSTANCE_GROUP_MANAGER = 'instance_group_manager'
 
+PROJECT_ID = "project-id"
+RESOURCE_ID = "resource-id"
+BODY = {"body": "test"}
+ZONE = "zone"
+SOURCE_INSTANCE_TEMPLATE = "source-instance-template"
+TIMEOUT = 120
+RETRY = mock.MagicMock(Retry)
+METADATA = [("key", "value")]
+
+GCP_CONN_ID = "test-conn"
+IMPERSONATION_CHAIN = ["ACCOUNT_1", "ACCOUNT_2", "ACCOUNT_3@google.com"]
+API_VERSION = 'v1'
+
+BASE_STRING = "airflow.providers.google.common.hooks.base_google.{}"
+COMPUTE_ENGINE_HOOK_PATH = "airflow.providers.google.cloud.hooks.compute.{}"
+
+
+class TestGcpComputeHookApiCall(TestCase):
```

Comment:
Since we're importing unittest (and using it as unittest.TestCase) we need to decide on either of the approaches (see what we're usually doing in other files): either use unittest.TestCase or from unittest import TestCase

And it's also recommended to use pytest classes for new test code whenenver possible, since we're adding new class it could be good time to recreate it in pytest format (most of it will look the same, setup mock will go to the top of class)

Reply:
Done